src/pages/scientificServices/pipelines/tabs/run/inputs/PipelineBooleanInput.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { LabeledCheckbox } from 'src/components/common';
+import { PipelineInput } from 'src/libs/ajax/teaspoons/teaspoons-models';
+import colors from 'src/libs/colors';
+
+interface PipelineBooleanInputProps {
+  input: PipelineInput;
+  value: boolean;
+  onChange: (value: boolean) => void;
+}
+
+export const PipelineBooleanInput: React.FC<PipelineBooleanInputProps> = ({ input, value, onChange }) => {
```

Comment:
is there a reasonable way to refactor some common stuff out between these different pipeline input type objects (are they called objects? variables?) or would that just become an unreadable mess?